### PR TITLE
Add `channel_opening_fees` column to `swaps_info` table

### PR DESCRIPTION
**Pull request by ok300**
- Implemented `FromSql` and `ToSql` traits for `OpeningFeeParams`
- Added `channel_opening_fees` column to `swaps_info` table, storing the params as json

### CODE REVIEW

**Review by erdemyerebasmaz — APPROVED** [review]
LGTM

File: libs/sdk-core/src/persist/swap.rs [hunk 1]
```diff
@@ -271,12 +273,7 @@ impl SqliteStorage {
             min_allowed_deposit: row.get("min_allowed_deposit")?,
             max_allowed_deposit: row.get("max_allowed_deposit")?,
             last_redeem_error: row.get("last_redeem_error")?,
-            min_msat: row.get("min_msat")?,
-            proportional: row.get("proportional")?,
-            valid_until: row.get("valid_until")?,
-            max_idle_time: row.get("max_idle_time")?,
-            max_client_to_self_delay: row.get("max_client_to_self_delay")?,
-            promise: row.get("promise")?,
+            channel_opening_fees: row.get("channel_opening_fees")?,
```
Comment (roeierez) [inline libs/sdk-core/src/persist/swap.rs:276]: How do we ensure backward computability here? Will deserialization it work for empty column?

**Review by roeierez — COMMENTED** [review]
Looks good. Dropped a question.

File: libs/sdk-core/src/persist/swap.rs (hunk 1 above)
Comment (ok300) [reply libs/sdk-core/src/persist/swap.rs:277]: Good point, I just added a test to check that:

https://github.com/breez/breez-sdk/pull/274/commits/0cf508ecb710c3e88cd0e02e2a517fadf7a09bdf#diff-cfb44f82cbd64bb9df0ce8fc3efd492f3341a5c7b700495cf113e1ef4618a3ceR420-R441

**Comment by ok300** [comment]
@ademar111190 do you know why the moonpay test sometimes fails? The error is: 

```
---- breez_services::tests::test_buy_bitcoin_with_moonpay stdout ----
create_test_persister /var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T//1504119542
Error: No LSP found for id ea51d025-042d-456c-8325-63e430797481
```

Is the LSP ID wrong?

**Review by roeierez — APPROVED** [review]

**Review by JssDWt — APPROVED** [review]

**Pull request merged**